### PR TITLE
PhpdocIndentFixer - fix bug when documentation is on the end of braces block

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php [hunk 1]
```diff
@@ -28,7 +28,12 @@ public function fix(\SplFileInfo $file, $content)
 
         foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
             $nextIndex = $tokens->getNextMeaningfulToken($index);
-            if (null === $nextIndex) {
+
+            // skip if there is no next token or if next token is block end `}`
+            if (
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php]: might look better on a single line

**Comment by keradus** [comment]
found on #902 regression

File: Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php (hunk 1 above)
Comment (keradus) [inline Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php]: good point, done

**Comment by keradus** [comment]
Thank you @keradus.

**Pull request closed without merging**